src/components/AboutMe.tsx

```diff
@@ -129,7 +130,7 @@ const RevatureAboutMe = () => {
 
     const handleSave = async () => {
         let portfolio = cookies['portfolio']
-        axios.post("http://3.236.213.150:8081/aboutMe", { portfolio, bio, email, phone})
+        axios.post(url + "/aboutMe", { portfolio, bio, email, phone})
```

Comment:
@ddinh could you please save `url + "/aboutMe/"` to a variable which you can pass into lines, 133, 149, 166, etc?  This is failing our code duplication quality checks.  Otherwise looks good!